docs/UserGuide.md

```diff
@@ -6,75 +6,136 @@ pageNav: 3
+
+The F.A.K.E.J.A.R.V.I.S. User Guide employs a variety of visual cues to enhance the information it presents. The table below offers an overview of the typographical conventions used.
+
+| Convention                                                                          | Description                                                     |
+|-------------------------------------------------------------------------------------|-----------------------------------------------------------------|
+| `Monospace`                                                                         | Used for command inputs, syntax, and file paths.                |
+| [Hyperlink](#)                                                                      | Indicates hyperlinks to external websites or within the guide.  |
+| **Bold text**                                                                       | Highlights important keywords.                                  |
+| **<div markdown="span" class="alert alert-info"> :information_source: Note </div>** | Provides information of special interest or importance.         |
+| **<div markdown="span" class="alert alert-warning"> :bangbang: Warning </div>**     | Alerts to potentially irreversible actions with data loss risk. |
+
+
+## Graphical User Interface
+
+![UI with Annotations](images/UiWithAnnotations.png)
+
+F.A.K.E.J.A.R.V.I.S.'s graphical user interface consists of 5 main components:
+* Command Box
+* Command Result Display
+* Task List Panel
+* Consultations Panel
+* Tutorials Panel
+
+You may enter your commands in the **Command Box** and then press Enter to execute them. The resulting message will be shown in the **Command Result Display** box.
+
+The following describes what each panel contains:
+* **Task List Panel:** Displays your set of tasks
+* **Consultations Panel:** Displays your consultations
+* **Tutorials Panel:** Displays your sessions
+
+### Task Card
+[Coming soon]
+
+### Consultation Card
+[Coming soon]
+
+### Session Card
+[Coming soon]
+
+
 --------------------------------------------------------------------------------------------------------------------
 
 ## Quick start
 
-1. Ensure you have Java `11` or above installed in your Computer.
+1. Ensure you have [Java 11](https://www.openlogic.com/openjdk-downloads?field_java_parent_version_target_id=406&field_operating_system_target_id=All&field_architecture_target_id=All&field_java_package_target_id=401) or above installed in your Computer.
 
-1. Download the latest `fakejarvis.jar` from [here](https://github.com/AY2324S1-CS2103T-T15-1/tp/releases). _[Coming Soon]_
+2. Download the latest `fakejarvis.jar` from [here](https://github.com/AY2324S1-CS2103T-T15-1/tp/releases). _[Coming Soon]_
 
-1. Copy the file to the folder you want to use as the _home folder_ for your F.A.K.E.J.A.R.V.I.S..
+3. Copy the file to the folder you want to use as the _home folder_ for your F.A.K.E.J.A.R.V.I.S..
```

Comment:
Not sure if the extra full stop at the end should be there for standardization since there already is a dot behind the 'S'.